assets/javascripts/discourse/connectors/above-main-container/custom-topic-lists-banner.gjs

```diff
@@ -11,17 +12,29 @@ export default class CustomTopicListsBanner extends Component {
       return;
     }
 
+    document.body.classList.add("category-header");
```

Comment:
I think you should use bodyClass for this - https://github.com/discourse/discourse/blob/67641340016528e6a46814aaf23de214a53f753f/plugins/chat/assets/javascripts/discourse/connectors/above-site-header/chat-body-class.gjs#L10